settings.lua

```diff
@@ -838,8 +838,7 @@ function courseplay:showFieldEdgePath(vehicle, pathType)
 end;
 
 function courseplay:changeDrawCourseMode(vehicle, changeBy)
-	vehicle.cp.drawCourseMode = courseplay:varLoop(vehicle.cp.drawCourseMode, changeBy, courseplay.COURSE_2D_DISPLAY_BOTH, courseplay.COURSE_2D_DISPLAY_OFF);
-	vehicle.cp.hud.changeDrawCourseModeButton:setActive(vehicle.cp.drawCourseMode ~= courseplay.COURSE_2D_DISPLAY_OFF);
```

Comment:
why did you remove this line?

Reply:
The complete function can be removed, as this is old code before the CourseDrawModeSetting was implemented and no longer used.